src/librustc/hir/map/definitions.rs

```diff
@@ -185,31 +185,31 @@ pub enum DefPathData {
     /// An impl
     Impl,
     /// Something in the type NS
-    TypeNs(InternedString),
+    TypeNs(Symbol),
```

Comment:
I think that these are `InternedString` so they hash their contents instead of the integer.